apps/datahub/src/app/organization/organization-details/organization-details.component.html

```diff
@@ -0,0 +1,164 @@
+<ng-container *ngrxLet="lastPublishedDatasets$ as lastPublishedDatasets">
+  <div id="organization-details-container" class="flex flex-col gap-[72px]">
+    <div class="h-full container-lg mx-auto px-[25px] w-full">
+      <div class="flex flex-row justify-between gap-[70px]">
+        <!------------------->
+        <!--  LEFT COLUMN  -->
+        <!------------------->
+        <div id="organization-details-left" class="w-2/3 mt-14">
+          <div class="flex flex-col gap-11">
+            <gn-ui-max-lines [maxLines]="2" *ngIf="organization.description">
+              <div>
+                <gn-ui-markdown-parser
+                  data-cy="organizationDescription"
+                  data-test="organizationDescription"
+                  [textContent]="organization.description"
+                ></gn-ui-markdown-parser>
+              </div>
+            </gn-ui-max-lines>
+          </div>
+        </div>
+
+        <!-------------------->
+        <!--  RIGHT COLUMN  -->
+        <!-------------------->
+        <div
+          id="organization-details-right"
+          class="w-[300px] flex flex-col gap-5"
+        >
+          <div
+            data-cy="organizationLogo"
+            class="bg-white border border-[#d8d8d8] w-full h-[185px] rounded-lg p-[30px] -mt-28"
```

Comment:
I'm ok to add this new color to the global variable.

But creating a css class for better readability ??? So are we dropping tailwind ? Because this IS the drawback of using tailwind, it creates big HTML, and to me this one is a tiny little one lol.

Reply:
yes, but in tailwind there is something called @apply that you may use in the .css files for better readability. Look up in the code, it's used several times. Also, I'm using this answer to signal that I didn't tell you that it's always good to use some proper tailwind classes. For instance h-[185px] could easily be h-48 (it's okay to round up - but up to your appreciation, I would do it personally). You can of course use actual values with the bracket method if the value you need is above what tailwind has in store.